docs/iris/src/whatsnew/latest.rst

```diff
@@ -10,406 +10,46 @@ This document explains the changes made to Iris for this release
 📢 Announcements
```

Comment:
Are you concerned about this passage in [release.rst](https://github.com/SciTools/iris/blob/89e880d7d7f76d1b77a635f152343c893698b5ff/docs/iris/src/developers_guide/release.rst)?

> Delete the ``docs/iris/src/whatsnew/latest.rst`` file so it will not cause an issue in the build

Reply:
@trexfeathers This is fine for `master`. Once I cut the `v3.0.x` branch, then I'll delete the `latest.rst` as it will no longer be relevant.